tests/lambda_functions/vetext_incoming_forwarder_lambda/test_vetext_incoming_forwarder_lambda.py

```diff
@@ -35,71 +48,113 @@ def test_verify_parsing_of_twilio_message(event):
     response = vetext_incoming_forwarder_lambda_handler(event, None)
 
     assert response['statusCode'] == 200
     sqs_mock.assert_called_once()
 
-
 @pytest.mark.parametrize('event', [(sqsInvokedWithAddOn)])
 def test_failed_sqs_invocation_call_throws_general_exception(mocker, event):
-    sqs_mock = mocker.patch(f'{lambda_module}.push_to_sqs')
-    mocker.patch(f'{lambda_module}.process_body_from_sqs_invocation', side_effect=Exception)
+    sqs_mock = mocker.patch(f'{LAMBDA_MODULE}.push_to_sqs')
+    mocker.patch(f'{LAMBDA_MODULE}.process_body_from_sqs_invocation', side_effect=Exception)
     response = vetext_incoming_forwarder_lambda_handler(event, None)
 
     assert response['statusCode'] == 500
     sqs_mock.assert_not_called()
 
-
 @pytest.mark.parametrize('event', [(albInvokedWithoutAddOn), (albInvokeWithAddOn)])
 def test_failed_alb_invocation_call_throws_general_exception(mocker, event):
-    sqs_mock = mocker.patch(f'{lambda_module}.push_to_sqs')
-    mocker.patch(f'{lambda_module}.process_body_from_alb_invocation', side_effect=Exception)
+    sqs_mock = mocker.patch(f'{LAMBDA_MODULE}.push_to_sqs')
+    mocker.patch(f'{LAMBDA_MODULE}.process_body_from_alb_invocation', side_effect=Exception)
     response = vetext_incoming_forwarder_lambda_handler(event, None)
 
     assert response['statusCode'] == 500
     sqs_mock.assert_not_called()
 
-
 @pytest.mark.parametrize('event', [(albInvokedWithoutAddOn), (albInvokeWithAddOn)])
 def test_failed_alb_invocation_call_throws_key_error_exception(mocker, event):
-    sqs_mock = mocker.patch(f'{lambda_module}.push_to_sqs')
-    mocker.patch(f'{lambda_module}.process_body_from_alb_invocation', side_effect=KeyError)
+    sqs_mock = mocker.patch(f'{LAMBDA_MODULE}.push_to_sqs')
+    mocker.patch(f'{LAMBDA_MODULE}.process_body_from_alb_invocation', side_effect=KeyError)
     response = vetext_incoming_forwarder_lambda_handler(event, None)
 
     assert response['statusCode'] == 424
     sqs_mock.assert_not_called()
+
+@pytest.mark.parametrize('event', [(albInvokedWithoutAddOn), (albInvokeWithAddOn), (sqsInvokedWithAddOn)])
+def test_failed_ssm_paramter_retrieval_throws_general_exception(mocker,  event):
+    sqs_mock = mocker.patch(f'{LAMBDA_MODULE}.push_to_sqs')
+    mocker.patch(f'{LAMBDA_MODULE}.read_from_ssm', return_value="")
+    response = vetext_incoming_forwarder_lambda_handler(event, None)
+
+    assert response['statusCode'] == 200
+    sqs_mock.assert_called_once()
+
+# GetEnv checks should go on queue
+@pytest.mark.parametrize('event', [(albInvokedWithoutAddOn), (albInvokeWithAddOn), (sqsInvokedWithAddOn)])
+def test_failed_getenv_vetext_api_endpoint_domain_property(mocker, event):
+    sqs_mock = mocker.patch(f'{LAMBDA_MODULE}.push_to_sqs')
+    mocker.patch(f'{LAMBDA_MODULE}.read_from_ssm', return_value="ssm")
+    mocker.patch.dict(os.environ, {"vetext_api_endpoint_path": "/some/path", "vetext_api_auth_ssm_path": "ssm"})    
```

Comment:
The code base has a fixture for overriding the environment that you could use.  See tests/conftest.py line 121 for the os_environ fixture.  Ditto for subsequent patches.

Reply:
This is fantastic feedback.  Thank you.  I had no idea what conftest.py was.